### PR TITLE
Issue #00 fix: variablizing keycloak heap memory size

### PR DESCRIPTION
**Pull request by harshavardhanc**

### CODE REVIEW

File: ansible/roles/keycloak-deploy/templates/standalone.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Uncomment the following line to prevent manipulation of JVM options
+# by shell scripts.
+#
+#PRESERVE_JAVA_OPTS=true
+
+#
+# Specify options to pass to the Java VM.
+#
+if [ "x$JAVA_OPTS" = "x" ]; then
+   JAVA_OPTS="-Xms64m -{{ keycloak_heap_size }} -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true"
```
Comment (keshavprasadms) [inline ansible/roles/keycloak-deploy/templates/standalone.conf.j2]: MaxMetaspaceSize=256m 
Provide a variable for this - Set default value to as 256 only.

File: ansible/roles/keycloak-deploy/defaults/main.yml [hunk 2]
```diff
@@ -77,6 +77,7 @@ keycloak_custom_ds: '
         </validation>
     </datasource>'
 
+keycloak_heap_size: Xmx1024m
```
Comment (keshavprasadms) [inline ansible/roles/keycloak-deploy/defaults/main.yml]: Change default to 512m

Comment (harshavardhanc) [reply ansible/roles/keycloak-deploy/defaults/main.yml]: Changes are updated

File: ansible/roles/keycloak-deploy/templates/standalone.conf.j2 (hunk 1 above)
Comment (harshavardhanc) [reply ansible/roles/keycloak-deploy/templates/standalone.conf.j2]: Changes are updated

File: ansible/roles/keycloak-deploy/defaults/main.yml [hunk 3]
```diff
@@ -77,6 +77,8 @@ keycloak_custom_ds: '
         </validation>
     </datasource>'
 
+keycloak_heap_size: Xmx512m
```
Comment (keshavprasadms) [inline ansible/roles/keycloak-deploy/defaults/main.yml]: Since we don't want users making typo - Can we make this 512m and then add Xmx in the template?

Comment (harshavardhanc) [reply ansible/roles/keycloak-deploy/defaults/main.yml]: Done

**Pull request merged**